src/components/ReactTags.js

```diff
@@ -341,14 +343,16 @@ class ReactTags extends Component {
   }
 
   moveTag(dragIndex, hoverIndex) {
-    const tags = this.props.tags;
+    if (this.props.allowDragDrop !== false) {
```

Comment:
@sp2hari this should be `!this.props.allowDragDrop` , seems like this got missed out, also why do we need this condition ? we are setting moveTag to null when `alloDragDrop` is false. can you verify this ?